src/plugins/plugin.datavalue.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+module.exports = function(Chart) {
+
+	var helpers = Chart.helpers;
+	// var noop = helpers.noop;
+	Chart.defaults.global.valueLabel = {
+		// position: 'top',
+		fontWeight: 'normal',
+		fontColor: '#000',
+		fontSize: 10,
+		padding: {top: 0, left: 0}
+		// truncate: true
+	};
+	function DrawValue(chart, element, value, index, chartype) {
+		var opts = chart.options.valueLabel;
+		var ctx = chart.ctx;
+		var position = element.getCenterPoint();
+		// print the value on the top of the bars.
+		if (chartype === 'bar') {
```

Comment:
This won't necessarily work for mixed chart types. It might be better to use 
`if (element instanceof Chart.elements.rectangle)`

I understand why there's a special case for bars, but it does seem a bit odd. @simonbrunel thoughts?